### PR TITLE
Android profile connections

### PR DESCRIPTION
**Pull request by MelihDemir99**

### CODE REVIEW

File: YallpAndroid/app/src/main/java/com/example/yallp_android/activities/GetStartedActivity.java [hunk 1]
```diff
@@ -83,22 +84,24 @@ public boolean isValidEmail(EditText editText){
...
-        progressDialog.setMessage("Please Wait");
+        progressDialog.setMessage("Please wait...");
         progressDialog.show();
 
         Call<Token> call;
 
         call = UserRetroClient.getInstance().getUserApi().signup(
                 new SignUpUser(username.getText().toString(),email.getText().toString(),password.getText().toString()));
-
+        Log.i("Melih_debug", call.toString());
```
Comment (fahricansanli) [inline YallpAndroid/app/src/main/java/com/example/yallp_android/activities/GetStartedActivity.java:94]: this one should be removed

File: YallpAndroid/app/src/main/java/com/example/yallp_android/activities/GetStartedActivity.java [hunk 2]
```diff
@@ -83,22 +84,24 @@ public boolean isValidEmail(EditText editText){
...
         call.enqueue(new Callback<Token>() {
             @Override
             public void onResponse(Call<Token> call, Response<Token> response) {
                 if(response.isSuccessful()){
+                    Log.i("Melih_debug", "success, token: " + response.body().getToken());
```
Comment (fahricansanli) [inline YallpAndroid/app/src/main/java/com/example/yallp_android/activities/GetStartedActivity.java:99]: another one

File: YallpAndroid/app/src/main/java/com/example/yallp_android/activities/GetStartedActivity.java [hunk 3]
```diff
@@ -111,7 +114,8 @@ public void onResponse(Call<Token> call, Response<Token> response) {
 
             @Override
             public void onFailure(Call<Token> call, Throwable t) {
-
+                Log.i("Melih_debug", t.getMessage());
```
Comment (fahricansanli) [inline YallpAndroid/app/src/main/java/com/example/yallp_android/activities/GetStartedActivity.java:117]: another one

File: YallpAndroid/app/src/main/java/com/example/yallp_android/activities/ProfileActivity.java [hunk 4]
```diff
@@ -2,30 +2,116 @@
...
+    private String userUsername = "default";
+    private String userMail = "default";
+    private String userName = "default";
+    private String userSurname = "default";
+    private String userBio = "default";
+
+
     @Override
     protected void onCreate(Bundle savedInstanceState) {
+        Log.i("Melih_debug", "got to profile page");
```
Comment (fahricansanli) [inline YallpAndroid/app/src/main/java/com/example/yallp_android/activities/ProfileActivity.java:40]: another one

File: YallpAndroid/app/src/main/java/com/example/yallp_android/activities/ProfileActivity.java [hunk 5]
```diff
@@ -2,30 +2,116 @@
...
         });
     }
+
+    public void getProfileInfo(final SharedPreferences sharedPref, final SharedPreferences.Editor editor){
+        if(sharedPref.getBoolean("newSession", false)){
+
+            Call<UserInfo> call;
+
+            String token = sharedPref.getString("token",null);
+            Log.i("Melih_debug", token);
```
Comment (fahricansanli) [inline YallpAndroid/app/src/main/java/com/example/yallp_android/activities/ProfileActivity.java:84]: another one

File: YallpAndroid/app/src/main/java/com/example/yallp_android/activities/ProfileActivity.java [hunk 6]
```diff
@@ -2,30 +2,116 @@
...
+                                .commit();
+                    }
+                    else{
+
+                    }
+                }
+
+                @Override
+                public void onFailure(Call<UserInfo> call, Throwable t) {
+                    Log.i("Melih_debug", "profile call didnt work: " + t.getMessage());
```
Comment (fahricansanli) [inline YallpAndroid/app/src/main/java/com/example/yallp_android/activities/ProfileActivity.java:108]: :)

**Review by sumeyrayilmaz-zz — APPROVED** [review]

**Pull request merged**